examples/CMakeLists.txt

```diff
@@ -39,7 +39,19 @@ if (${CHECKPOINT_BUILD_EXAMPLES})
 
     target_link_libraries(
       ${EXAMPLE}
+      PUBLIC
```

Comment:
I had to add this so CMake wouldn't complain about mixing plain and keyword syntax. It's in a linked example, so I think it's no big deal how we do it.